### ISSUE TITLE
Feedback for “Memgraph direct download links”

### ISSUE THREAD

**Issue by mfanara** [memgraph/documentation#1074]
Downloading memgraph_2.22.0-1_amd64.deb to install on Ubuntu 24.04.1 LTS.

I try to install as follows, but this deb file wants an older version of libpython

sudo apt install ./memgraph_2.22.0-1_amd64.deb 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'memgraph' instead of './memgraph_2.22.0-1_amd64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 memgraph : Depends: libpython3.9 (>= 3.9.1) but it is not installable
            Depends: libssl1.1 (>= 1.1.0) but it is not installable
E: Unable to correct problems, you have held broken packages.

**Comment by katarinasupe** [comment]
Can you try [downloading here](https://memgraph.com/download) as well and let me know if it works as expected?

**Comment by mfanara** [comment]
Yes that works fine - it is just too big for my edge application.

What I am trying to accomplish is the creation of a docker container image built from a debian slim image where I simply do a apt install <memgraph.deb> in the docker build.

I can do that without any problem

I also need MAGE in the image. That is where I am stuck. I tried simply copying the query_modules folder from the memgraph-platform container image into my custom docker image at container build time but that didn't work. memgraph didn't recognize the existence of the additional modules.

I have a number of open issues in memgraph and memgraph-mage related to this.

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

**Comment by katarinasupe** [comment]
Can you please link the issues you created here as well?
